sdk/storage/perf-tests/storage-blob-track-1/test/upload.spec.ts

```diff
@@ -4,12 +4,7 @@
 import { generateUuid } from "@azure/core-http";
```

Comment:
doesn't the perf-stress package depend directly on uuid? why do we need this?

Reply:
Good catch, can be removed.

Reply:
Done.